### ISSUE TITLE
Toolkit has a command for local debug of .net lambda functions for CodeLenses to execute

### ISSUE THREAD

**Issue by mpiroc** [aws/aws-toolkit-vscode#514]
Follow-up task to #289. Acceptance criteria:

* In addition to the 'run locally' codelens, .NET Core lambda handlers should have a 'debug locally' codelens.
* When invoked, this will perform the same actions as the 'run locally' command, with the following differences:
  * Build in the 'Debug' configuration
  * Pass a debug port to SAM CLI.
  * Attach the debugger once SAM CLI output indicates that it is ready for the debugger to attach.
